server/sonar-web/src/main/js/app/components/help/TutorialsHelp.js

```diff
@@ -21,10 +21,12 @@
 import React from 'react';
 import { translate } from '../../../helpers/l10n';
 
+/*::
 type Props = { onTutorialSelect: () => void };
+*/
 
-export default function TutorialsHelp({ onTutorialSelect }: Props) {
-  const handleClick = (event: Event) => {
+export default function TutorialsHelp({ onTutorialSelect } /*: Props */) {
+  const handleClick = (event /*: Event */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)